WooCommerce/Resources/en.lproj/Localizable.strings

```diff
@@ -6231,6 +6231,9 @@ which should be translated separately and considered part of this sentence. */
 /* A 'Learn More' label text, which shows tax information upon being clicked. */
 "order.form.paymentSection.taxes.learnMore" = "Learn More.";
 
+/* Label for the row showing the shipping tax. */
```

Comment:
I was reading https://github.com/woocommerce/woocommerce-ios/blob/trunk/docs/localization.md and I am not sure if we also add this here or the release process adds it here and then we upload it for translation?